core/renderers/block_rendering_rewrite/block_render_draw.js

```diff
@@ -198,8 +198,9 @@ Blockly.blockRendering.Drawer.prototype.drawStatementInput_ = function(row) {
   if (this.highlighter_) {
     this.highlighter_.drawStatementInput(row);
   }
+  var input = row.getLastInput();
   // Where to start drawing the notch, which is on the right side in LTR.
-  var x = row.xPos + row.statementEdge +
+  var x = row.xPos + input.xPos +
```

Comment:
`input.xPos` should include `row.xPos`.

This probably shows up on the block with an output and a statement input, since that shifts the row's xPos.

Reply:
Done